test/vignette/http/legacy/routes_test.clj

```diff
@@ -243,7 +243,7 @@
                              (request :get "/intmap_tile_set_4823/thumb/20150205173220%21phpZDfa00.jpg/1110x300x5-20150205173220%21phpZDfa00.jpg")))]
     (:width map) => "1110"
     (:height map) => "300"
-    (:original map) => "20150205173220!phpZDfa00.jpg"
+    (:original map) => "20150205173220%21phpZDfa00.jpg"
```

Comment:
I thought about adding tests to make sure that we eventually end up with a proper url-decoded path, but I don't know if it makes sense for us to unit test compojure. It has a [good amount of those](https://github.com/weavejester/compojure/tree/master/test/compojure).